### DISCUSSION TITLE
Fonts are missing on every system start

### DISCUSSION THREAD

**Discussion by buehler** [tonsky/FiraCode#1472] [Q&A] [answered]
I do know that this probably has nothing to do with the font itself. But I'm kind of lost right now.
I found the article in the Wiki that some Windows 11 users need to "unblock" each file before installing. Otherwise, the font is "not available" to the system.

I did that, and it worked like a charm. However, now I need to install the font on every system restart. The font is lost after a while, and I need to reinstall it (the unblocked version).

Did someone experience the same behavior?

I'm using the newest Windows 11 build and Windows Terminal / VS Code on Windows.

Regards

**Comment by kenmcd** [comment]
Where are the installed fonts located?
In the Windows Fonts folder or in your User Fonts folder?
How did you install the fonts? (exact steps)

**Reply by buehler** [reply]
I installed them like follows:
- downloaded the release from this repository
- extracted all files
- rightclicked each ttf (firacode light to bold, and retina) and "unlocked" them
- marked all files -> rightclick -> install

When I go to the fonts control panel and get the properties of the font, the install location shows:
`C:\Users\<<USER>>\AppData\Local\Microsoft\Windows\Fonts`

**Comment by kenmcd** [comment, marked as answer]
When you right-click on the selected fonts is the only option "Install" ? 
Is "Install for all users" not available?
If that is not available it means your user does not have administrator rights.
Just "Install" will install the fonts in your user fonts folder (which is the location you showed).
Some applications have font access problems with User fonts. 
Switch to an administrator user (if possible) and install the fonts with right-click and select "Install for all users".
That will install the fonts in the Windows Fonts folder and should avoid the owner/permissions/blocked issues.

**Reply by buehler** [reply]
Hey @kenmcd 
Thanks for your suggestions. I need to confirm if the Font has vanished again. I'm definitely the administrator of the computer and the Fonts are installed in the Userfolder. It seems strange that the font just vanish although they are listed in the Fonts folder. When the issue happens, Terminal and VSCode do not recognize the font, but in the Fonts folder, they are still present.

I'll try the install for all users if the issue happens again.

**Reply by buehler** [reply]
alright, it seems that "install for all users" did the trick. Somehow Terminal and VSCode kind of lost the access to the font when it's only installed for one user. Don't know why 🤷🏾
